README.md

```diff
@@ -65,6 +67,43 @@ echo 'GITHUB_TOKEN=<your api token>' >> ~/.config/bonfire/env
 
 **Please note**: When using the Github token (classic), it **must have** the `read:project` scope for the template fetching to work!
 
+## Using Docker
+
+We provide a Docker image at quay.io/cloudservices/bonfire. The image wraps a Python virtual environment with **crc-bonfire** installed on it.
+
+In order to use Bonfire you have to provide either a valid KUBE_CONFIG or a URL and a TOKEN to connect to the target Openshift cluster.
+
+The container's entrypoint already invokes the **bonfire** CLI with the arguments passed to the container, so you should be able to run any bonfire command you'd typically run by using:
+
+```
+podman run -it --rm quay.io/cloudservices/bonfire version
+```
+
+### Using credentials
+
+The Bonfire image can use the `OC_LOGIN_SERVER` and `OC_LOGIN_TOKEN` variables to connect to an Openshift cluster.
+
+```
+podman run -it --rm -e OC_LOGIN_SERVER=https://api.openshift.example.com:6443 \
+    -e OC_LOGIN_TOKEN=some-api-token \
+    quay.io/cloudservices/bonfire namespace list
+```
+
+It can also make use of a Kube config file by mounting it on the container's HOME:
+
+```
+podman run -it --rm -v ${KUBECONFIG:="$HOME/.kube/config"}:/opt/bonfire/.kube/config:Z,U \
```

Comment:
Needs the `--userns` arg to ensure local kubeconfig permissions don't get altered after the container runs:

```suggestion
podman run -it --rm --userns=keep-id:uid=1000,gid=1000 -v ${KUBECONFIG:="$HOME/.kube/config"}:/opt/bonfire/.kube/config:Z,U \
```